manifests/zone.pp

```diff
@@ -58,6 +58,10 @@
 #   An array of IP addresses from which queries should be allowed
 #  Defaults to an empty array, which allows all ip to query the zone
 #
+# [*allow_update*]
+#   An array of IP addresses from which updates should be allowed
+#  Defaults to an empty array, which denies updates to the zone
```

Comment:
This needs a warning, as it will probably surprise people that when they enable this, no other records for this zone will take effect.